11 - Custom Video Player/index.html

```diff
@@ -8,7 +8,7 @@
 <body>
 
    <div class="player">
-     <video class="player__video viewer" src="652333414.mp4"></video>
+     <video class="player__video viewer" src="652333414.mp4" ></video>
```

Comment:
:)
why add the space?
be aware that such small changes pollute your merge request. this may water down your merge request when you have bigger changes.